docs/overview/component/index.md

```diff
@@ -23,77 +12,10 @@ This document will give you an overview of all the components developed and used
   - [Vald Agent Scheduler](#vald-agent-scheduler)
   - [Vald Index Manager](#vald-index-manager)
 - [Vald Replication Manager](#vald-replication-manager)
-  - [Vald Replication Manager Agent](#vald-replication-manager-agent)
-  - [Vald Replication Manager Controller](#vald-replication-manager-controller)
 - [Kubernetes Components](#kubernetes-components)
   - [Kube-apiserver](#kube-apiserver)
   - [Custom Resources](#custom-resources)
 
-## Vald Filter
-
-Vald Filter is an optional functionality in Vald.
-User can implement the custom filtering logic and integrate with Vald.
-
-Vald Filter provides the following functionalities.
-
-- Custom filter based on request query
-- Custom filter for the searching result
-
-### Vald Ingress Filter
-
-Vald Ingress Filter filters the incoming request before processing it.
-
-Users can implement custom filtering logic such as changing the vectors or filtering based on user ID.
-
-### Vald Egress Filter
-
-Vald Egress Filter filters the response before sending it to the user.
-
-This component can reorder the searching result from multiple Vald Agents based on the user-defined ranking.
-
-### Vald Filter Gateway
-
-Vald Filter Gateway forwards the request to Vald Ingress Filter before processing it and forwards the response to the Vald Egress Filter before returning the searching result to the user.
```

Comment:
Since we have decided to separate filter gateway to another repository when we discuss about simple Vald, I suppose it will be removed soon from this repository, so I removed here.

Should we remove it from here?